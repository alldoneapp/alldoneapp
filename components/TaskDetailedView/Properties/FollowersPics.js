import React, { Component } from 'react'
import { Image, StyleSheet, TouchableOpacity } from 'react-native'

export default class FollowersPics extends Component {
    constructor(props) {
        super(props)
    }

    render() {
        return (
            <TouchableOpacity style={localStyles.container} onPress={this.onPress}>
                <Image style={localStyles.userImage} source={{ uri: this.props.photoURL }}></Image>
            </TouchableOpacity>
        )
    }

    onPress = () => {}
}

const localStyles = StyleSheet.create({
    container: {
        flexDirection: 'row',
        alignItems: 'center',
        paddingHorizontal: 16,
        paddingVertical: 8,
    },
    userImage: {
        width: 32,
        height: 32,
        backgroundColor: 'transparent',
        marginRight: 4,
    },
})
