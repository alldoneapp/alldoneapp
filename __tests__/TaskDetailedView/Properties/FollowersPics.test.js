import React from 'react'
import FollowersPics from '../../../components/TaskDetailedView/Properties/FollowersPics'

import renderer from 'react-test-renderer'

describe('FollowersPics component', () => {
    describe('FollowersPics snapshot test', () => {
        it('should render correctly', () => {
            const tree = renderer.create(<FollowersPics />).toJSON()
            expect(tree).toMatchSnapshot()
        })
    })
})
