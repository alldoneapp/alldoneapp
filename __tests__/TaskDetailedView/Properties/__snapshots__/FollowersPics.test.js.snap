// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FollowersPics component FollowersPics snapshot test should render correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "opacity": 1,
      "paddingHorizontal": 16,
      "paddingVertical": 8,
    }
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "backgroundColor": "transparent",
        "height": 32,
        "marginRight": 4,
        "width": 32,
      }
    }
  />
</View>
`;
